index.js

```diff
@@ -330,6 +330,36 @@ walletRouter.post('/melt-tokens', (req, res) => {
   }
 });
 
+/**
+ * GET request to get utxo details
+ * For the docs, see api-docs.js
+ */
+walletRouter.get('/utxo-details', (req, res) => {
```

Comment:
I'd call it just `GET /utxo-filter`. Isn't it more intuitive?

Reply:
Done